third-party/realdds/src/dds-device-server.cpp

```diff
@@ -117,11 +120,11 @@ static void on_discovery_stream_header( std::shared_ptr< dds_stream_server > con
         { "stream-name", stream->name() },
         { "options" , stream_options },
         { "intrinsics" , intrinsics },
-        { "recommended-filters", stream_filters },
+        { "recommended-filters", std::move( stream_filters ) },
```

Comment:
Won't move be better also for intrinsics and options?

Reply:
You could do it for everything, but really the biggest value would be for compound json objects or objects that allocated memory of their own. I don't believe intrinsics fit, but you're right about options.

Reply:
Changed. Will be in next PR.